src/Main/Main.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
```

Comment:
While it might be useful to disable a specific eslint rule in some cases, it's generally not a good practice. The rule 'jsx-a11y/control-has-associated-label' is important for accessibility. All form controls used should have associated text labels.

Reply:
this has been fixed. Thank you. I often use autoformatting and there are such possibilities